# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/WebTaskScheduling.webidl
# Commit: bffd879a7547
# Full Hash: bffd879a75476ea682e86082f960e24884d6ca2e
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-04-01 09:40:15
# Regressor Bug: 1920115
# File Overlap Count: 2
# Description:
#   Bug 1920115 - Implement scheduler.yield r=jjaschke,dom-core,webidl,smaug
#   
#   Spec: https://wicg.github.io/scheduling-apis/#dom-scheduler-yield
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D234232
# ==============================================================================

diff -r f441cb5e7e6e -r bffd879a7547 dom/webidl/WebTaskScheduling.webidl
--- a/dom/webidl/WebTaskScheduling.webidl	Mon Mar 31 19:49:02 2025 +0000
+++ b/dom/webidl/WebTaskScheduling.webidl	Mon Mar 31 19:49:02 2025 +0000
@@ -31,6 +31,9 @@
     SchedulerPostTaskCallback callback,
     optional SchedulerPostTaskOptions options = {}
   );
+
+  [BinaryName="yieldImpl"]
+  Promise<undefined> yield();
 };
 
 dictionary TaskControllerInit {